### PR TITLE
Prepare 1.1.0 release

### PR DESCRIPTION
**Pull request by jaelim-ms**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,28 @@
 # App Center SDK for iOS and macOS Change Log
 
+## Version 1.1.0
+
+This version contains a bug fix and improvements.
+
+### AppCenter
+
+* **[Fix]** Fix a locale issue that doesn't properly report system locale if an application doesn't support current language.
+* **[Improvement]** Change log level to make HTTP failures more visible, and add more logs.
```
Comment (clpolet) [inline CHANGELOG.md]: You probably don't need the comma as you have an `and`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
+
+### MobileCenterDistribute
+
+* **[Improvement]** Add Portuguese to supported languages, see [this folder](https://github.com/Microsoft/mobile-center-sdk-ios/tree/develop/MobileCenterDistribute/MobileCenterDistribute/Resources) for a list of supported languages.
+* **[Improvement]** Automatically migrate secret values that were stored in pre-GA versions.
```
Comment (clpolet) [inline CHANGELOG.md]: I think it's not explicit enough, it should explain what's the impact for the customers. Like: `Apps integrating Mobile Center can now upgrade to another version integrating App Center without the need to reinstall.`. Or something similar.

File: CHANGELOG.md (hunk 1 above)
Comment (clpolet) [reply CHANGELOG.md]: Nevermind it works.

File: CHANGELOG.md (hunk 2 above)
Comment (jaelim-ms) [reply CHANGELOG.md]: Revised.

**Pull request merged**